internal/abi/fastly/types.go

```diff
@@ -884,6 +884,14 @@ func (b *BackendConfigOptions) SNIHostname(sniHostname string) {
 	b.opts.sniHostnameLen = prim.U32(buf.Len())
 }
 
+func (b *BackendConfigOptions) PoolConnections(poolingOn bool) {
+	if poolingOn {
+		b.mask &^= backendConfigOptionsMaskDontPool
```

Comment:
It's a pity the ABI defined the flag in the negative :(

Reply:
This was intentional, so that folks using older SDKs would default to using pooling, which most people want.